### PR TITLE
Implement new login tests

### PR DESCRIPTION
**Pull request by viorelaioia-zz**

### CODE REVIEW

File: tests/test_login_to_rps.py [hunk 1]
```diff
@@ -98,3 +114,32 @@ def test_login_servicenow_stage(self, selenium, ldap, urls, counter_api):
                                                                        conftest.increase_otp_counter(counter_api)))
         assert test.is_profile_icon_displayed
         test.logout()
+
+    def test_login_biztera(self, selenium, ldap, urls, counter_api):
```
Comment (johngian) [inline tests/test_login_to_rps.py]: All three following methods are doing the same thing. Please deduplicate the code and override appropriately.

File: pages/selfservice_mig.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+from pages.base import Base
+from pages.two_factor_authentication import TwoFactorAuthentication
+
+
+class SelfServiceMig(Base):
+    def __init__(self, selenium, url):
+        super(SelfServiceMig, self).__init__(selenium)
+        self.go_to_url(url)
+
+    def login_with_ldap(self, email, password):
```
Comment (johngian) [inline pages/selfservice_mig.py]: ``login_with_ldap`` and ``__init__`` are exactly the same in both classes defined.

File: pages/testrp.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+from pages.auth0 import Auth0
+from pages.base import Base
+from pages.two_factor_authentication import TwoFactorAuthentication
+
+
+class TestRp(Base):
+    _sign_in_button = (By.CSS_SELECTOR, '.btn-signin.btn-signin-red')
+    _logout_button_locator = (By.CSS_SELECTOR, '#main-content a[href="/logout"]')
+
+    def __init__(self, selenium, url):
```
Comment (johngian) [inline pages/testrp.py]: Same here

File: pages/testrp.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        super(TestRp, self).__init__(selenium)
+        self.go_to_url(url)
+
+    @property
+    def is_logout_button_displayed(self):
+        return self.is_element_visible(*self._logout_button_locator)
+
+    def login_with_ldap(self, email, password):
```
Comment (johngian) [inline pages/testrp.py]: Same here.

**Review by johngian — CHANGES_REQUESTED** [review]

**Comment by viorelaioia-zz** [comment]
@johngian , PR is updated. Can you take another look? Thanks!

**Review by johngian — APPROVED** [review]

**Pull request merged**